test/ApiCheck.Test/ApiListingComparerTests.cs

```diff
@@ -80,6 +79,68 @@ public void Compare_Detects_TypeGenericAritybreakingChanges_as_removal()
             Assert.Contains(expected, breakingChanges);
         }
 
+        [Theory]
+        [InlineData("public class ComparisonScenarios.ClassToRemoveFieldsFrom")]
+        [InlineData("public struct ComparisonScenarios.StructToRemoveFieldsFrom")]
+        public void Compare_DetectsAllFieldRemovals(string typeToCheck)
+        {
+            // Arrange
+            var v1ApiListing = CreateApiListingDocument(V1Assembly);
+            var v2ApiListing = CreateApiListingDocument(V2Assembly);
+            var comparer = new ApiListingComparer(v1ApiListing, v2ApiListing);
+
+            // Oops. The NewInternalProperty addition is a breaking change; makes it impossible to subclass type in
+            // another assembly.
```

Comment:
API check doesn't complain when an `internal abstract` member is added though that is a significant breaking change -- prevents creation of a subclass outside the defining assembly. As I said, it's a corner case but "oops", something is slipping though&hellip;